backend/src/config/index.ts

```diff
@@ -128,8 +128,9 @@ if (
 
 const federation = {
   FEDERATION_BACKEND_SEND_ON_API: process.env.FEDERATION_BACKEND_SEND_ON_API ?? '1_0',
+  // ?? operator don't work here as expected
   FEDERATION_VALIDATE_COMMUNITY_TIMER:
-    Number(process.env.FEDERATION_VALIDATE_COMMUNITY_TIMER) ?? 60000,
+    Number(process.env.FEDERATION_VALIDATE_COMMUNITY_TIMER) || 60000,
```

Comment:
```suggestion
    Number(process.env.FEDERATION_VALIDATE_COMMUNITY_TIMER ?? 60000) ,
```

perhaps the operator will have not the expected effect, because the property is evaluated inside the invocation of Number( ), right?

Reply:
you are right, yes ok.